index.Rmd

```diff
@@ -286,7 +290,7 @@ In order to achieve these objectives, you should try to follow a few good practi
     - Consistent placement of curly braces.
 * [Make your own packages](#__optional__Writing_packages_).
 * Keep a similar [directory structure](../RProjectManagement/index.html) across your projects.
```

Comment:
It is unclear which part of tutorial 1 is relevant for keeping a consistent directory structure. Perhaps the link is not necessary.